hedera-node/hedera-app/src/test/java/com/hedera/node/app/components/IngestComponentTest.java

```diff
@@ -47,7 +52,12 @@ class IngestComponentTest {
 
     @BeforeEach
     void setUp() {
-        given(platform.getCryptography()).willReturn(cryptography);
+        Configuration configuration = new TestConfigBuilder().getOrCreateConfig();
+        PlatformContext platformContext = mock(PlatformContext.class);
+        when(platformContext.getConfiguration()).thenReturn(configuration);
+        when(platform.getContext()).thenReturn(platformContext);
+
+        given(platform.getContext().getCryptography()).willReturn(cryptography);
```

Comment:
While `given(platform.getContext().getCryptography())` does the same thing as `given(platformContext.getCryptography())`, it seems unwise to do this because the `platform` mock will  increment the number of times `platform.getContext()` was called even though it was only called while setting up another mock. Call count isn't checked here, but does seem like a risky habit to get into. 
```suggestion
        given(platformContext.getCryptography()).willReturn(cryptography);
```